docs/src/app/components/pages/customization/themes.jsx

```diff
@@ -56,8 +55,8 @@ const ThemesPage = React.createClass({
   },
 
   getStyles() {
-    let canvasColor = this.context.muiTheme.rawTheme.palette.canvasColor;
-    let borderColor = this.context.muiTheme.rawTheme.palette.borderColor;
+    let canvasColor = this.props.muiTheme.rawTheme.palette.canvasColor;
```

Comment:
I've been swapping `rawTheme` for `baseTheme` as I've been seeing it. Should we do that now?